### ISSUE TITLE
Leading spaces in sample dovecot init.d script

### ISSUE THREAD

**Issue by awesterb** [dovecot/documentation#357]
The sample dovecot init.d script listed in the [documentation](https://doc.dovecot.org/installation_guide/startup_scripts/sample_dovecot_init.d_script/) reads
```
 ### BEGIN INIT INFO
 # Provides:          dovecot
 [...]
```
instead of
```
### BEGIN INIT INFO
# Provides:          dovecot
[...]
```
Starting the init info block with a whitespace is not permitted by [the specification](https://refspecs.linuxbase.org/LSB_3.0.0/LSB-PDA/LSB-PDA/initscrcomconv.html) (as it mentions "All lines inside the [init] block shall begin with a hash character '#' in the first column".)

At any rate, `update-rc.d dovecot defaults` creates no symlinks in `/etc/rc*.d` when the leading whitespaces are present.

**Comment by sirainen** [comment]
Fix in https://github.com/dovecot/documentation/pull/358

**Comment by cmouse** [comment]
Closed via #358
